templates/c2s-pharmacy-finder/1/rancher-compose.yml

```diff
@@ -2,7 +2,8 @@ version: '2'
 
 catalog:
   name: nhsuk_c2s

```

Comment:
The invalid chars were added previously. I've removed them locally but the changes haven't been pushed for some reason.